src/main/java/edu/escuelaing/ieti/matchmaking/exception/UserMatchmakingException.java

```diff
@@ -0,0 +1,13 @@
+package edu.escuelaing.ieti.matchmaking.exception;
+
+public class UserMatchmakingException extends MatchmakingException{
```

Comment:
if `MatchmakingException` extended from `RuntimeException` you wouldn't need to add the `UserMatchmakingException`  in the throws clause of any method